packages/keystone/bin/commands/start.js

```diff
@@ -0,0 +1,28 @@
+const keystone = require('@keystone-alpha/core');
+const { executeDefaultServer, getEntryFileFullPath } = require('../utils');
+
+module.exports = {
+  // prettier-ignore
+  spec: {
+    '--port':   Number,
+    '-p':       '--port',
+    '--entry':  String,
+    '--out':    String,
+    '-o':       '--out',
+  },
+  help: ({ exeName }) => `
+    Usage
+      $ ${exeName} start --port=3000
+
+    Options
+      --port, -p  Port to start on [${keystone.DEFAULT_PORT}]
+      --entry     Entry file exporting keystone instance [${keystone.DEFAULT_ENTRY}]
+      --out, -o   Directory where build was saved [dist]
```

Comment:
🤔 Is `--out` potentially confusing here? Perhaps it should be `--build` or `--dist`?

Or maybe not an option at all, like next's CLI:

```
next start --help

    Description
      Starts the application in production mode.
      The application should be compiled with `next build` first.

    Usage
      $ next start <dir> -p <port>

    <dir> is the directory that contains the compiled dist folder
    created by running `next build`.
    If no directory is provided, the current directory will be assumed.
    You can set a custom dist folder in config https://github.com/zeit/next.js#custom-configuration

    Options
      --port, -p      A port number on which to start the application
      --hostname, -H  Hostname on which to start the application
      --help, -h      Displays this message
```

I like their `<dir>` option which doesn't need a command line switch.